packages/paste-core/components/spinner/src/keyframes.tsx

```diff
@@ -0,0 +1,50 @@
+import {keyframes} from '@twilio-paste/styling-library';
+
+// const originalKeyframes = `
```

Comment:
FYI these are the original keyframes. Check out [this codesandbox](https://codesandbox.io/s/keyframe-exploration-4r6yj) to see the visual difference.